root/programs/C#/Frameworks/Infrastructure/Public/IO/SymmetricCryptography.cs

```diff
@@ -438,9 +438,9 @@ private static SymmetricAlgorithm CreateSymmetricAlgorithm(EnumSymmetricAlgorith
         {
             SymmetricAlgorithm sa = null;
 
-            // AesCryptoServiceProvider, AesManagedは3.5からの提供。
-            // 暗号化プロバイダ選択のポリシー（Managed → CSP → CNG）
-            // に合わせて、AesCryptoServiceProviderを削除。
+            // AesCryptoServiceProvider, AesManagedは.NET Framework 3.5からの提供。
+            // 暗号化プロバイダ選択の優先順は、Managed → CAPI(CSP) → CNG。
```

Comment:
@yusukemaegawa 

There are features to develop immediately with RSACryptoServiceProvider.
However, RSACryptoServiceProvider has not been CNG compliant yet.

I am aware of the change in the priority of CAPI and CNG, but I'd like you to merge for now for the following reasons.

- It takes a lot of time to change from CAPI to CNG.
- Development of the above functions has priority.

For coping with changes in priorities of CAPI and CNG, they will be registered as issues.